src/Uno.UI.Composition/Composition/CompositionSurfaceBrush.skia.cs

```diff
@@ -77,16 +77,16 @@ private static bool TryGetSkiaCompositionSurface(ICompositionSurface? surface, [
 
 		internal override void UpdatePaint(SKPaint fillPaint, SKRect bounds)
 		{
-			if (TryGetSkiaCompositionSurface(Surface, out var scs))
+			if (TryGetSkiaCompositionSurface(Surface, out var scs) && scs is { Image: SKImage image })
```

Comment:
If I'm right, the `Image` can be `null` anymore, you should just make the `Image` non nullable and remove the check here.